rmgpy/data/solvation.py

```diff
@@ -802,12 +802,6 @@ def estimateSoluteViaGroupAdditivity(self, molecule):
         addedToPairs = {} # Dictionary of key = atom, value = # lone pairs changed
         saturatedStruct = molecule.copy(deep=True)
 
-        # Convert lone pairs to radicals, then saturate with H.
-       
-        # Change lone pairs to radicals based on valency
-        if sum([atom.lonePairs for atom in saturatedStruct.atoms]) > 0: # molecule contains lone pairs
-            saturatedStruct, addedToPairs = self.transformLonePairs(saturatedStruct)
```

Comment:
Yes, I don't really agree with the approach suggested there, and it is not incompatible with the way we have set up new trees. I will check that the solvation trees don't give horrible estimates after this change, might be another commit added to database PR if necessary.

Reply:
Leaving the `transformLonePairs()` we removed here causes the following error on this branch:
```
======================================================================
ERROR: Test that we can obtain solute parameters via group additivity for CO.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/alongd/Code/RMG-Py/rmgpy/data/solvationTest.py", line 181, in testSoluteDataGenerationCO
    soluteData = self.database.getSoluteDataFromGroups(species)
  File "/home/alongd/Code/RMG-Py/rmgpy/data/solvation.py", line 685, in getSoluteDataFromGroups
    sdata = self.estimateSoluteViaGroupAdditivity(molecule)
  File "/home/alongd/Code/RMG-Py/rmgpy/data/solvation.py", line 823, in estimateSoluteViaGroupAdditivity
    self.__addGroupSoluteData(soluteData, self.groups['abraham'], saturatedStruct, {'*':atom})
  File "/home/alongd/Code/RMG-Py/rmgpy/data/solvation.py", line 859, in __addGroupSoluteData
    raise KeyError('Node has no parent with data in database.')
KeyError: 'Node has no parent with data in database.'
```
Where `[C-]#[O+]` is being erroneously converted into:
```
<Molecule "C=O">
1 C u0 p0 c-1 {2,T} {3,S} {4,S}
2 O u0 p1 c+1 {1,T}
3 H u0 p0 c0 {1,S}
4 H u0 p0 c0 {1,S}
```
So obviously this process of handling carbenes does not treat CO correctly.
We could either stay with the present commit deleting these lines, or add a condition that this will not be applied for the `C2tc` atomType.